packages/css/src/components/button/button.scss

```diff
@@ -3,42 +3,96 @@
  * Copyright (c) 2023 Gemeente Amsterdam
  */
 
-@import "../../../node_modules/@utrecht/components/button/css";
-
 @mixin reset {
   -webkit-text-size-adjust: 100%;
 }
 
 .amsterdam-button {
+  border: none;
+  cursor: var(--amsterdam-action-activate-cursor);
```

Comment:
We shouldn't import common level tokens directly in the css. The component level tokens should import the brand/common level ones.

That way, external users only have to override the component level tokens to get their styling.